index.bs

```diff
@@ -399,11 +401,14 @@ like
     get <a href="#rs-locked">locked</a>()
 
     <a href="#rs-cancel">cancel</a>(reason)
-    <a href="#rs-get-reader">getReader</a>()
+    <a href="#rs-get-iterator">getIterator</a>({ preventCancel } = {})
+    <a href="#rs-get-reader">getReader</a>({ mode } = {})
```

Comment:
Drive-by fix: `getReader()` was missing its `{ mode }` argument in the class definition.

If you prefer I put this in a separate PR, let me know.